source/docs/03-metadata.xml

```diff
@@ -1444,6 +1444,82 @@
+                     <p>It is recommended to use at least the elements <gi scheme="MEI">title</gi>, <gi scheme="MEI">date</gi>, <gi scheme="MEI">annot</gi> and <gi scheme="MEI">locus</gi> in the description:</p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample080.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p>For a detailed description or encoding of a watermark, respectively, <gi scheme="MEI">fig</gi> can be used, which on the one hand allows to refer to an existing image (<gi scheme="MEI">graphic</gi>) of the watermark and on the other hand increases the depth of the description. To mark up different components of a watermark, several <gi scheme="MEI">heraldry</gi> elements can be used in a <gi scheme="MEI">figDesc</gi> and related to each other by means of <gi scheme="MEI">relation</gi>.
+                     </p>
+                     <p>
+                        <figure>
+                           <head/>
+                           <egXML xmlns="http://www.tei-c.org/ns/Examples" rend="code" xml:space="preserve"><xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="../examples/header/header-sample081.txt" parse="text"/></egXML>
+                        </figure>
+                     </p>
+                     <p>To ensure that the description of the pictorial signs conforms to international standards and that the individual components of the watermark are correctly represented, the multilingual description catalogue of the <ref target="https://www.memoryofpaper.eu/BernsteinPortal/" type="_blank">Bernstein project</ref> or the <ref target="http://www.paperhistory.org/Standards/" type="_blank">IPH standard</ref> should be consulted. How to refer to already existing databases with watermarks, see <ptr target="#linkingdata"/>Linked Data (13.)
+                     </p>
+                     <p><specList>
+                        <specDesc key="stamp"/>
+                     </specList></p>
+                     <p>Stamps can appear in many forms in manuscripts and prints, for example as library stamps, library signatures, postmarks, ownership marks, address stamps or legal notices. The description of the stamp therefore depends on the individual, project-specific requirements: However, it is helpful to first consider whether to merely name the occurring stamps, describe their textual elements as well as their form in detail, or refer to a graphic with the facsimile of the stamp.</p>
```

Comment:
```suggestion
                     <p>Stamps can appear in many forms in manuscripts and prints, for example as library stamps, library signatures, postmarks, ownership marks, address stamps or legal notices. The description of the stamp therefore depends on individual, project-specific requirements. However, it is helpful to first consider whether it is sufficient to merely name the occurring stamps, or if is also desirable to record their form and textual elements in detail, or to refer to a graphic showing a facsimile of the stamp.</p>
```

Reply:
Looks fine. @gucl-mu any suggestions?

Reply:
OK wait. I think I misunderstood what Clemens meant here.

Does "naming" occurring stamps mean to "describe their textual elements as well as their form in detail" ?? 

Then let's use parentheses to disambiguate. 
"However, it is helpful to first consider whether to merely name the occurring stamps (describing their textual elements as well as their form in detail), or refer to a graphic with the facsimile of the stamp."

If, however, these are three different options
1. name the stamp
2. record form and textual elements
3. refer to a graphic showing a facsimile

Then:
"However, it is helpful to first consider whether it is sufficient to merely name the occurring stamps or if is also desirable to record their form and textual elements in detail, or to refer to a graphic showing a facsimile of the stamp."
Note that in this version, it is like the one I originally suggested, but I removed a comma, and inserted a missing "it".

Reply:
Option B it is. So three variants: Naming, describing or illustrating. Your suggestion is good, let's take it